ui/app/styles/utils/_bulma_variables.scss

```diff
@@ -61,6 +61,10 @@ $radius: 2px;
 //box
 $box-radius: 0;
 $box-shadow: 0 0 0 1px rgba($black, 0.1);
+$box-shadow-low: 0 5px 1px -2px rgba($black, 0.12), 0 3px 2px -1px rgba($black, 0);
+$box-shadow-middle: 0 8px 4px -4px rgba($black, 0.10), 0 6px 8px -2px rgba($black, 0.05);
+$box-shadow-high: 0 12px 5px -7px rgba($black, 0.08), 0 11px 10px -3px rgba($black, 0.10);
+$box-shadow-highest: 0 16px 6px -10px rgba($black, 0.06), 0 16px 16px -4px rgba($black, 0.20);
```

Comment:
Would it make sense to make these a "scale" and just order them numerically (box-shadow-N)? Mostly in case we ever want to add more levels in the future.

Reply:
I don't think so, mostly because a user can't really distinguish very many more anyway. We already got rid of "lowest" because it just wasn't needed, so I think we'll stick to these four and I think the friendly names help understand what they're doing.